pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -42,7 +41,13 @@ For the very best experience with all Code.org content, we recommend consulting
 | **To use Web Lab** | `https://downloads.computinginthecore.org`<br/>`https://codeprojects.org`                                                                              |
 
 ## Mobile Support Details
-Our Hour of Code activities and CS Fundamentals courses can be used on mobile devices. While these activities should be usable, they are not optimized for a mobile experience and have some known limitations (e.g. touch targets may be small, accessory elements may display offscreen, and text may be difficult to input using the on-screen keyboard). Note: the CS Discoveries, CS Principles, and Intro to App Lab courses are not usable on mobile devices.
+All CS Discoveries, CS Principles, and Intro to App Lab courses are not compatible with tablets or mobile devices - a full keyboard and mouse (or touchpad, like on Macbooks) are required. Surface Pros with a keyboard attachment and Chromebooks are compatible. iPads or other tablets with a keyboard attachment are not compatible.
```

Comment:
Nit: MacBooks has a capital b.

Reply:
Do we need to specify the "Microsoft" Surface Pro? I think Microsoft is in the name?

Reply:
Can we just specify 'tablets' instead of 'iPads or other tablets'?

Reply:
Normally these edits are made by content editors, so git doesn't have a history regarding why we explicitly specify iPads. I'm guessing in the past we got lots of questions specific to iPads, so we called out iPads specifically here. I think it makes sense to keep that here. I'll go ahead and make the other two changes you suggested.